src/coreclr/gc/gc.cpp

```diff
@@ -48405,6 +48800,10 @@ HRESULT GCHeap::Initialize()
             // This needs to be different from our initial heap count so we can make sure we wait for
             // the idle threads correctly in gc_thread_function.
             gc_heap::dynamic_heap_count_data.last_n_heaps = 0;
+            // This should be adjusted based on the target tcp. See comments in gcpriv.h
+            gc_heap::dynamic_heap_count_data.below_target_threshold = 10.0;
```

Comment:
can these be constants with some descriptive names for clarity?

Reply:
do you have suggestions for more descriptive names? these names were supposed to be descriptive, or so I thought. their purpose is explained in the header (as the comment says).

Reply:
ok missed the comment. This is good for now.